src/app/core/services/user/user.service.ts

```diff
@@ -25,6 +25,7 @@ import { ApiService, AvailableOptions } from 'ish-core/services/api/api.service'
 interface CreatePrivateCustomerType extends CustomerData {
   address: Address;
   credentials: Credentials;
+  user?: User;
```

Comment:
Any idea how I could get rid of this not correct addition to satisfy this line?
`{ ...newCustomer, user: { ...newCustomer.user, preferredLanguage: currentLocale.lang ?? 'en_US' } }`
Does this change reflect the idea you suggested?